pkg/webhook/policy.go

```diff
@@ -118,6 +131,17 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		return vResp
 	}
 
+	// default response is deny
+	requestResponse := denyResponse
```

Comment:
@maxsmythe thanks for ping, missed this (and couldn't find the comment now). what do you think of default response as deny? it should be overwritten later with correct requestResponse

Reply:
We may want to create an "unknown" type so that we know there is a problem, otherwise if we're adding an "allow" logic path, the metrics might misreport as "deny"